addons/website/static/src/scss/secondary_variables.scss

```diff
@@ -103,6 +103,9 @@ $o-has-customized-colors:
     or map-get($o-user-theme-color-palette, 'warning')
     or map-get($o-user-theme-color-palette, 'danger'));
 
+$gray-hue: o-website-value('gray-hue');
+$gray-extra-saturation: o-website-value('gray-extra-saturation');
```

Comment:
Add the `!default` flag

Reply:
The value of `$gray-huy `and `$gray-extra-saturation` must be synchronized with the values chosen by the user, IMO the value of the two variables must always be replaced by the new ones (so no !default).

Reply:
It makes sense indeed :+1: